content/en/agent/kubernetes/operator_configuration.md

```diff
@@ -28,550 +28,550 @@ spec:
 
 `agent.systemProbe.securityContext.windowsOptions.runAsUserName`
-: Use the `UserName` in Windows to run the entry point of the container process. Defaults to the user specified in image metadata if unspecified. May also be set in `PodSecurityContext`. If set in both `SecurityContext` and `PodSecurityContext`, the value specified in `SecurityContext` takes precedence. This field is beta-level and may be disabled with the `WindowsRunAsUserName` feature flag.                  
+: Use the `UserName` in Windows to run the entry point of the container process. Defaults to the user specified in image metadata if unspecified. May also be set in `PodSecurityContext`. If set in both `SecurityContext` and `PodSecurityContext`, the value specified in `SecurityContext` takes precedence. This field is beta-level and may be disabled with the `WindowsRunAsUserName` feature flag.
 
 `agent.useExtendedDaemonset`
-: Use ExtendedDaemonset for Agent deployment. Default value is false.           
+: Use ExtendedDaemonset for Agent deployment. Default value is false.
 
 `clusterAgent.additionalAnnotations`
-: `AdditionalAnnotations` provide annotations that are added to the Cluster Agent Pods.            
+: `AdditionalAnnotations` provide annotations that are added to the Cluster Agent Pods.
 
 `clusterAgent.additionalLabels`
-: `AdditionalLabels` provide labels that are added to the cluster checks runner Pods. 
+: `AdditionalLabels` provide labels that are added to the cluster checks runner Pods.
 
 `clusterAgent.affinity.nodeAffinity.preferredDuringSchedulingIgnoredDuringExecution`
-: The scheduler will prefer to schedule pods to nodes that satisfy the affinity expressions specified by this field, but it may choose a node that violates one or more of the expressions. The node that is most preferred is the one with the greatest sum of weights. That is, for each node that meets all of the scheduling requirements (resource request, `requiredDuringScheduling` affinity expressions, etc.), compute a sum by iterating through the elements of this field and adding "weight" to the sum if the node matches the corresponding `matchExpressions`; the node(s) with the highest sum are the most preferred.     
+: The scheduler will prefer to schedule pods to nodes that satisfy the affinity expressions specified by this field, but it may choose a node that violates one or more of the expressions. The node that is most preferred is the one with the greatest sum of weights. That is, for each node that meets all of the scheduling requirements (resource request, `requiredDuringScheduling` affinity expressions, etc.), compute a sum by iterating through the elements of this field and adding "weight" to the sum if the node matches the corresponding `matchExpressions`; the node(s) with the highest sum are the most preferred.
```

Comment:
```suggestion
: The scheduler prefers to schedule pods to nodes that satisfy the affinity expressions specified by this field, but it may choose a node that violates one or more of the expressions. The node that is most preferred is the one with the greatest sum of weights. For each node that meets all of the scheduling requirements (resource request and `requiredDuringScheduling` affinity expressions), compute a sum by iterating through the elements of this field and adding "weight" to the sum if the node matches the corresponding `matchExpressions`; the node(s) with the highest sum are the most preferred.
```